### ISSUE TITLE
Better Support for Touch for Mute Hot Keys

### ISSUE THREAD

**Issue by owlboy** [matzman666/OpenVR-AdvancedSettings#71]
I'm not sure if you have access to hardware for testing. But if it could be made so the face buttons (ABXY) on Touch controllers could be used with the mute feature, that would be fantastic!

Thanks for the tools!

**Comment by xxTheGoDxx** [comment]
Agreed, most games (probably especially Steam VR games) don't use the face buttons and I know none that uses all of them.
